#1239 allow multiple repos in FileMonitorConfiguration.

ask4gilles opened the pull request:
Fixes gh-1239

spencergibb commented on spring-cloud-config-monitor/src/main/java/org/springframework/cloud/config/monitor/FileMonitorConfiguration.java:
```diff
@@ -185,17 +186,19 @@ public void poll() {
+					repositoryUri = repository.getUri();
+					Resource resource = this.resourceLoader.getResource(repositoryUri);
+					if (resource instanceof FileSystemResource) {
+						return Collections.singleton(Paths.get(resource.getURI()));
```
I guess you only watch the first one? What if there are multiple `file://` git repos?

ask4gilles commented:
Hi @spencergibb ,
Thanks for looking at this.
I've corrected the behaviour in case there are multiple paths and I've also added a unit test.
I don't see any IT tests in this module though...
